seacrowd/sea_datasets/wisesight_thai_sentiment/wisesight_thai_sentiment.py

```diff
@@ -0,0 +1,185 @@
+  url          = {https://doi.org/10.5281/zenodo.3457447}
+}
+"""
+
+
+_DATASETNAME = "wisesight_thai_sentiment"
+
+
+
+_DESCRIPTION = """\
+Wisesight Sentiment Corpus: Social media messages in Thai language with sentiment category (positive, neutral, negative, question)
+* Released to public domain under Creative Commons Zero v1.0 Universal license.
+* Category (Labels): {"pos": 0, "neu": 1, "neg": 2, "q": 3}
+* Size: 26,737 messages
+* Language: Central Thai
+* Style: Informal and conversational. With some news headlines and advertisement.
+* Time period: Around 2016 to early 2019. With small amount from other period.
+* Domains: Mixed. Majority are consumer products and services (restaurants, cosmetics, drinks, car, hotels), with some current affairs.
+* Privacy:
+    * Only messages that made available to the public on the internet (websites, blogs, social network sites).
+    * For Facebook, this means the public comments (everyone can see) that made on a public page.
+    * Private/protected messages and messages in groups, chat, and inbox are not included.
+* Alternations and modifications:
+    * Keep in mind that this corpus does not statistically represent anything in the language register.
+    * Large amount of messages are not in their original form. Personal data are removed or masked.
+    * Duplicated, leading, and trailing whitespaces are removed. Other punctuations, symbols, and emojis are kept intact.
+    (Mis)spellings are kept intact.
+    * Messages longer than 2,000 characters are removed.
+    * Long non-Thai messages are removed. Duplicated message (exact match) are removed.
+* More characteristics of the data can be explore: https://github.com/PyThaiNLP/wisesight-sentiment/blob/master/exploration.ipynb
+"""
+
+_SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
+_UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
+
+_HOMEPAGE = "https://github.com/PyThaiNLP/wisesight-sentiment"
+
+_LANGUAGES = ["tha"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+
+_LICENSE = Licenses.CC0_1_0.value
+
+
+_LOCAL = False
+
+
+_URLS = {
+    _DATASETNAME: "https://github.com/PyThaiNLP/wisesight-sentiment/raw/master/huggingface/data.zip",
+}
+
+# TODO: add supported task by dataset. One dataset may support multiple tasks
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]  # example: [Tasks.TRANSLATION, Tasks.NAMED_ENTITY_RECOGNITION, Tasks.RELATION_EXTRACTION]
+
+# TODO: set this to a version that is associated with the dataset. if none exists use "1.0.0"
+#  This version doesn't have to be consistent with semantic versioning. Anything that is
+#  provided by the original dataset as a version goes.
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class WisesightSentimentDataset(datasets.GeneratorBasedBuilder):
+    """Wisesight Sentiment Corpus: Social media messages in Thai language with sentiment category (positive, neutral, negative, question)"""
+
+    _DOWNLOAD_URL = _URLS["_DATASETNAME"]
```

Comment:
the `_DATASETNAME` should be using variable and not the constant
```suggestion
    _DOWNLOAD_URL = _URLS[_DATASETNAME]
```

Reply:
Oh yep sorry